op-service/eth/supervisor_status.go

```diff
@@ -0,0 +1,11 @@
+package eth
+
+// SupervisorChainStatus is the status of a chain as seen by the supervisor.
+type SupervisorChainStatus struct {
+	// LocalDerived is the latest L2 block that the chain was derived from.
+	LocalDerived BlockRef `json:"localDerived"`
+	// LocalDerivedFrom is the origin of LocalDerived.
+	LocalDerivedFrom L1BlockRef `json:"localDerivedFrom"`
+	// LocalUnsafe is the latest L2 block that has been processed by the supervisor.
+	LocalUnsafe BlockRef `json:"localUnsafe"`
```

Comment:
I don't think this gives us what we need.  I'm not sure if `LocalDerivedFrom` is actually the L1 origin of the L2 block, or if it's the L1 block that contained the last piece of batch data required to derive that L2 block.

Neither is what challenger needs though - it needs to know the latest L1 block that has been fully processed for all chains, even if it didn't contain any batch data or cause any change to any chain. Challenger needs to know that L1 block was considered and there was no action required, because challenger waits until the game's L1 head has been processed before playing the game since then it knows any data relevant to the game has been processed by the node.

Reply:
`LocalDerivedFrom` is just the L1 origin of the L2 block. Though sometimes it's identical to the `CurrentL1` in the op-node sync status, which sounds like what we want, because it uses the same event as `CurrentL1` updates.
Will fix.

Reply:
CurrentL1 is what we want, but that is _not_ the L1 origin of the current L2 head block in op-node. At least, it shouldn't be and if it is we broke something.  We actually use `CurrentL1 - 1` in challenger because `CurrentL1` isn't guaranteed to be fully processed but op-node only looks at one L1 block at a time, so it's parent must have been fully processed.

https://github.com/ethereum-optimism/optimism/blob/910c9ade39c0bcdff5f2badd94efbe016a428e73/op-service/eth/sync_status.go#L6-L10

Reply:
This is fixed. The L1 head watermark across all chains replaces `LocalDerived` and `LocalDerivedFrom` in the sync status.